db/migrate/20170403153216_create_links.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLinks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :links do |t|
+      t.string :uri
```

Comment:
I don't think it does because Postgres varchar doesn't have a limit like MySQL does